Assets/Scripts/UI/InteractTrigger.cs

```diff
@@ -17,7 +17,7 @@ void Update()
 
     void OnTriggerStay2D(Collider2D col)
     {
-        if (col.gameObject.tag == "Player")
+        if (col.gameObject.tag == "Furniture")
```

Comment:
can you just check whether the layer is Interactive? that way you don't have to make new tag